app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes_counter + 1)
+  end
```

Comment:
- [ ] It would be nice if we use one of [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), the `after_save` method to call the private method `update_likes_counter`. Let use kindly make the method private

Reply:
Thanks @addod19 for your insightful review. Will work on changes right away